index.html

```diff
@@ -25,6 +26,14 @@
         <li><a href="#about-section">About</a></li>
         <li><a href="#contact">Contact</a></li>
       </ul>
+      <div id="theMenu">
+        <ul class="nav-items2">
+            <li><a class='navs' href="#work-section">Portfolio</a></li>
+            <li><a class='navs' href="#about-section">About</a></li>
+            <li><a class='navs' href="#contact">Contact</a></li>
```

Comment:
- [ Optional] Nice one reducing the font 👏 . But I strongly suggest that you heed the request of the previous reviewer to add more margin/padding to the top of the menu link since it is very close to the (X) button. This gives a more clear and appealing look and structure, and also helps you stick to the design 👍 

# View:
![image](https://user-images.githubusercontent.com/50990925/217129507-3c0cdf23-92db-46da-8045-c103ed1862f3.png)

# Figma:
![image](https://user-images.githubusercontent.com/50990925/217129557-d021ae05-8729-41f8-b37e-26b386ebd079.png)